guide/payments/request.md

```diff
@@ -82,31 +70,35 @@ Below is an overview of these various entry points:
 
 The most common way bitcoin is requested today is in the form of single-use payment requests. For Lightning, a standard [BOLT11](https://github.com/lightning/bolts/blob/master/11-payment-encoding.md) invoice is used. For on-chain, [addresses of various types]({{ "/guide/glossary/address/" | relative_url }}) are used. Addresses can be reused though this is not advised due to [privacy concerns]({{ "/guide/payments/privacy/#generate-a-new-address-for-each-payment" | relative_url }}). Single-use invoices and addresses need to be easily and conveniently generated within a wallet application.
 
-### Amount
+As single-use payment requests are convenient for one-time or in-person payments, generating them should be possible from your applications home screen in a fast, convenient, and, private manner.
 
-When generating a single-use payment request, the receiver may want to add an amount to be paid. Entering an amount from the home screen can speed up the requesting flow. Alternatively, users can add an amount to an already generated payment request. Entering an amount needs to be clear to prevent users entering incorrect amounts.
+Do not show balances on your home screen when generating single-use payment requests as a third party, especially if requesting in person, is likely to see the users balance which is a privacy concern. Home screens can also enter users straight into a request flow, such as entering an amount to be requested, speeding up the process.
```

Comment:
Let's move up the home screen example that's further down the page. There are several references in the copy to UI techniques in that example that people will not understand when reading from top-to-bottom.